src/docbkx/chapter91-monitor.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<chapter xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xmlns="http://www.oasis-open.org/docbook/4.5" id="embedding_datacleaner"
+		 xsi:schemaLocation="http://www.oasis-open.org/docbook/4.5 http://www.oasis-open.org/docbook/xsd/4.5/docbook.xsd">
+
+	<title>DataCleaner monitor</title>
+
+	<chapterinfo>
+		<abstract>
+			<para>
+				Following sections describe known issues you may encounter while using
+				DataCleaner Monitor. You will also find a workaround or tips how to
+				fix the problem when such a solution exists.
+			</para>
```

Comment:
Can we use plural here ? Such as "workarounds", "problems", "solutions".  After "tips" word "for" is missing